src/tokens/TabWallet.tsx

```diff
@@ -153,6 +161,26 @@ function TabWallet({ navigation, route }: Props) {
     navigation.setParams({ activeAssetTab: selectedTab })
   }
 
+  if (tokens.length === 0) {
```

Comment:
unlikely, but what if the user has positions but no tokens?

Reply:
I was thinking about this. Most likely any position they have will be in the home/earn view and they can interact with it there. Also, Kayla's made it clear that this will be redone before a public release. Not perfect but i think it will suffice for now